Lib/netrc.py

```diff
@@ -23,10 +23,7 @@ class netrc:
     def __init__(self, file=None):
         default_netrc = file is None
         if file is None:
-            try:
-                file = os.path.join(os.environ['HOME'], ".netrc")
-            except KeyError:
-                raise OSError("Could not find .netrc: $HOME is not set") from None
+            file = os.path.expanduser("~/.netrc")
```

Comment:
`'/'` is Posix specific pathname separator. Windows accepts it too, but it is better to use the native pathname separator, `os.sep`. Or use `os.path.join()`:
```python
os.path.expanduser(os.path.join("~", ".netrc"))
```
or
```python
os.path.join(os.path.expanduser("~"), ".netrc")
```